Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -430,13 +423,20 @@ def scan_result_command(client: Client, args: Dict[str, Any], api_key) -> List[C
                         score=score
                     )
 
-                    ip_indicator = Common.IP(
-                        ip=qid_data['data']['indicator'],
-                        asn=qid_data['data']['properties']['geo'].get('asn'),
-                        geo_country=qid_data['data']['properties']['geo'].get('country'),
-                        port=qid_data.get('data', {}).get('attributes', {}).get('port'),
-                        dbot_score=dbot_score
-                    )
+                    if 'geo' in qid_data['data']['properties']:
+                        ip_indicator = Common.IP(
+                            ip=qid_data['data']['indicator'],
+                            asn=qid_data['data']['properties']['geo'].get('asn'),
+                            geo_country=qid_data['data']['properties']['geo'].get('country'),
+                            port=qid_data.get('data', {}).get('attributes', {}).get(
+                                'port') if qid_data['data']['attributes'] != [] else None,
+                            dbot_score=dbot_score
+                        )
+                    else:
+                        ip_indicator = Common.IP(
+                            ip=qid_data['data']['indicator'],
+                            dbot_score=dbot_score
```

Comment:
Is there a reason not to add `port` as well?

Reply:
It's not always there, and I need to test it first.
I will do this in the next update. This one is now pretty good tested.